Cargo.toml

```diff
@@ -35,6 +35,7 @@ lazy_static = "1.4.0"
 base64 = "0.21.4"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+serde-wasm-bindgen = "0.4"
```

Comment:
why this change?

Reply:
I use it's `from_value` function to parse the object argument [here](https://github.com/embroider-build/content-tag/pull/62/files#diff-c030113407a164cbd1ad3500c613f59abd4346ba13ee5b653c56f393c937b0a8R15)

Reply:
is it a successor to regular wasm-bindgen? or are the non-serde deps just on their way out anyway?

Reply:
![image](https://github.com/embroider-build/content-tag/assets/60734210/42788ecc-3158-49d7-a9e2-db9254ca463b)
I got this from the docs, that's why I used it https://docs.rs/serde-wasm-bindgen/latest/serde_wasm_bindgen/.

Reply:
tbh I didn't really have time to learn wasm or rust enough to properly understand what I was doing

Reply:
oooo that's nice, thanks for providing that info!